goshopify.go

```diff
@@ -201,11 +203,19 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 		return err
 	}
 
+	bodyBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return err
+	}
```

Comment:
Why did you make changes to this file? This seems to be unrelated to the main PR, so could you revert it for now?